pkg/minikube/driver/driver.go

```diff
@@ -198,17 +198,30 @@ func Suggest(options []registry.DriverState) (registry.DriverState, []registry.D
 	}
 
 	alternates := []registry.DriverState{}
+	rejects := []registry.DriverState{}
 	for _, ds := range options {
 		if ds != pick {
-			if !ds.State.Healthy || !ds.State.Installed {
+			glog.Errorf("%s: %s", ds.Name, ds.Rejection)
```

Comment:
I feel like all this logic could be done in one loop instead of two, though I'm not sure if there's a real advantage of either strategy.

Otherwise this PR is great.

Reply:
I agree, but I wasn't smart enough to figure out how without making it overly complicated.

We aren't sure who to pick until the end of the first loop - so no matter what, we need to iterate twice to part out the alternates. I'l leave this for someone better at reasoning about algorithms.